lib/pos_snapshot_entries.go

```diff
@@ -966,6 +966,37 @@ func (bav *UtxoView) GetSnapshotLeaderScheduleValidator(leaderIndex uint16) (*Va
 	return validatorEntry, nil
 }
 
+func (bav *UtxoView) GetSnapshotLeaderSchedule() ([]*PKID, error) {
```

Comment:
The code itself looks fine. Can you add a simple unit test for this in `TestRunEpochCompleteHook` to verify? Take a look at the usage of the function `GetSnapshotLeaderScheduleValidator` in the end of epoch hook test